Fix issue #103

kalletlak opened the pull request.

zhx828 commented on app/scripts/vueCore.js:
```diff
@@ -131,6 +133,114 @@
+            openCases:function(){
+              iViz.openCases();
+              
+             /* var studyId = '';
```
Can we remove these commented out code? I think they have been isolated into an individual function?

zhx828 commented on app/scripts/vueCore.js:
```diff
@@ -262,6 +372,21 @@
+              position: {my:'bottom center', at:'top center', viewport: $(window)}
+            });
+            $('#study-view-header-left-6').qtip({
+              prerender: true,
```
Why prerender the qtip?

zhx828 commented on app/styles/partials/_global.scss:
```diff
@@ -72,14 +72,14 @@
 }
 
 #study-view-header-right-1 {
```
Should we get rid of 'study-view' name?

kalletlak committed:
Fix vue initialization and query form issue

kalletlak committed:
Change logic in updating invisible charts

kalletlak committed:
Code Cleanup

1. Get rid of bridge chart, now just use the dimension.
2. Use invisible dimension to update special charts.
3. Update sync logic between groups.
4. Remove unwanted code.

kalletlak committed:
Fix pie chart qtip table loading

kalletlak committed:
Fixed issues

1. Fixed loading piechart qtip datatable
2. Fixed issue while clearing all filters
3. Replace #.each with _.each
4. Fixed issue while refreshing filtered scatter plot

kalletlak committed:
Small fixes

1. Rename invisibleChart to invisibleDimension in special charts.
2. Replace remaining $.each to _.each

kalletlak committed:
Changing sync logic between groups

kalletlak committed:
More changes

1. Replace invisible chart with just dimension.
2. Add dimension for table and scatter plots.
3. Update logic behind syncing chart groups.
4. Show loading image for special charts when filters are changed.

zhx828 committed:
Merge pull request #146 from kalletlak/performance

Performance

kalletlak committed:
Remove session-module related files

zhx828 committed:
Merge pull request #145 from kalletlak/multi-cohort-cleanup

Remove session-module related files

Commit:
Merge branch 'multi-cohort' into HEAD

Conflicts:
	app/index.html

kalletlak committed:
Fix issue #103

kalletlak committed:
Name changes and bower dc.js dependency fix

kalletlak committed:
Fix style
